boilerplates/sentry.py

```diff
@@ -41,9 +45,17 @@ class Sentry:
         sentry_sdk.integrations.threading.ThreadingIntegration()
     ]
 
+    debug: bool = False
+    attach_stacktrace: bool = False
+    shutdown_timeout: float = 2.0
```

Comment:
https://docs.sentry.io/platforms/python/configuration/options/#shutdown_timeout

according to Sentry docs, the timeout variable type is `int` -- so not sure if specifying decimal places would even work.